onnx/optimizer/passes/fuse_consecutive_squeezes.h

```diff
@@ -0,0 +1,55 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
+namespace ONNX_NAMESPACE {
+namespace optimization {
+
+struct FuseConsecutiveSqueezes final : public OptimizePass {
+  explicit FuseConsecutiveSqueezes()
+      : OptimizePass("fuse_consecutive_squeezes", API_TYPE::IR) {}
+
+  // returns a vector `ret` such that squeeze by `ret` is equivalent
+  // to squeeze by `t1` and then by `t2`
+  std::vector<int64_t> compose_squeezes(
+      const std::vector<int64_t>& t1,
+      const std::vector<int64_t>& t2) {
+    std::vector<int64_t> ret;
+    ret.reserve(t1.size() + t2.size());
```

Comment:
The axes computation is probelmatic, such as axes=(0,2) and axes=(0,) in squeeze 1 and 2, we will get wrong axes in the fused op

Reply:
The method still cannot handle all the cases. For example, squeeze1 with axes=(0,1,2,4) and squeeze2 with axes=(0,). Since axes should be a small vector, you can create a mapping from squeeze2 dims to squeeze1 dims.

Also squeeze1’s axes should be sorted.